swaglyrics/__main__.py

```diff
@@ -10,63 +10,84 @@
 
 
 def main():
-	# 	print(r"""
-	#  ____                     _               _
-	# / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
-	# \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
-	#  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
-	# |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
-	#                     |___/      |___/
-	# 	""")
-	print('Updating unsupported.txt from server.')
-	with open('unsupported.txt', 'w', encoding='utf-8') as f:
-		response = requests.get('http://aadibajpai.pythonanywhere.com/master_unsupported')
-		f.write(response.text)
-	print("Updated unsupported.txt successfully.")
+    # 	print(r"""
```

Comment:
Why is the indentation changed?

Reply:
I used autopep8 to correct that over-indent error. I was reading about pycodestyle, that is, pep8 python module, in that autopep8 was also given. pycodestyle gave all the errors and warnings, and by using autopep8 it solved all of them.